airbyte-integrations/connectors/source-surveymonkey/source_surveymonkey/streams.py

```diff
@@ -20,23 +20,13 @@ class SurveymonkeyStream(HttpStream, ABC):
     url_base = "https://api.surveymonkey.com/v3/"
     primary_key = "id"
     data_field = "data"
+    default_backoff_time: int = 60  # secs
 
     def __init__(self, start_date: pendulum.datetime, survey_ids: List[str], **kwargs):
         super().__init__(**kwargs)
         self._start_date = start_date
         self._survey_ids = survey_ids
 
-    def backoff_time(self, response: requests.Response) -> Optional[float]:
-        """
-        # respecting daily limit if valid
-            daily_limit_remaining = int(response.headers.get('X-Ratelimit-App-Global-Day-Remaining'))
-            if daily_limit_remaining and daily_limit_remaining<=0:
-                return response.headers.get('X-Ratelimit-App-Global-Day-Reset')
-        """
-        minute_limit_remaining = int(response.headers.get("X-Ratelimit-App-Global-Minute-Remaining"))
-        if minute_limit_remaining and minute_limit_remaining <= 1:
```

Comment:
backoff does not work when minute_limit_remaining becomes = 0, then condition does not work